DTypes 01 – Strings/DTYPES01-005.js

```diff
@@ -0,0 +1,13 @@
+/*
+DTYPES01-005: Define a string variable with following format:
+My name’s <name>. I’m <age> years old.
+Previously define the name and the age values. They will not be changed during the 
+further code.
+*/
+
+const firstName = 'Svilen';
+const age = '40';
+
+let stringVar = `My name’s ${firstName}. I’m ${age} years old.`;
```

Comment:
Using the apostrpophe will print the following results.

<img width="347" alt="image" src="https://github.com/karastoyanov/JSF-DIW/assets/86146020/8e119a07-91b9-4d24-afb0-2d790fb71a0c">


You can replace `My name’s` with `My name's`. However, the solution is correct.